Xamarin.Essentials/Types/PlacemarkExtensions.shared.cs

```diff
@@ -1,6 +1,33 @@
-﻿namespace Xamarin.Essentials
+﻿using System;
+
+namespace Xamarin.Essentials
 {
-    internal static partial class PlacemarkExtensions
+#pragma warning disable SA1205 // Partial elements should declare access
```

Comment:
@mattleibow do we need this disable? how should we do this?

Reply:
I believe we do. If you try it it will complain either way. The only way to get it to compile was by adding this pramga